components/WarningNote/WarningNotes.tsx

```diff
@@ -49,9 +49,9 @@ export const WarningBox = ({ notes, personId }: Props): React.ReactElement => {
                     </dd>
```

Comment:
probably the role on this file needs to be changed from `alert` to `complementary`.

alerts are more for content that is dynamic and caused by user action: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role#accessibility_concerns

Reply:
Done in https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/357/commits/5ac4d6fd7746ee54e6c3239c18aa0135555be897